### PR TITLE
docs: document copa github action

### PR DESCRIPTION
**Pull request by ashnamehrotra**
Signed-off-by: ashnamehrotra <ashnamehrotra@gmail.com>

Describe the changes in this pull request using active verbs such as _Add_, _Remove_, _Replace_ ...

Closes https://github.com/project-copacetic/copa-action/issues/4

### CODE REVIEW

File: docs/github-action.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copa Github Action
```
Comment (sozercan) [inline docs/github-action.md:1]: should we move this to website so it's more visible? wdyt?

File: docs/github-action.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
```
Comment (sozercan) [inline docs/github-action.md:2]: please add an example here

File: docs/github-action.md (hunk 1 above)
Comment (ashnamehrotra) [reply docs/github-action.md:1]: yes! moved

Comment (sozercan) [reply docs/github-action.md]: please make sure to update sidebar

File: website/docs/github-action.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+          id: vuln_count
+          run: |
+            report_file="report.json"
+            vuln_count=$(jq '.Results | length' "$report_file")
+            echo "vuln_count=$vuln_count" >> $GITHUB_OUTPUT
+
+        - name: Copa Action
+          if: steps.vuln_count.outputs.vuln_count != '0'
+          id: copa
+          uses: project-copacetic/copa-action@v1.0.0
```
Comment (sozercan) [inline website/docs/github-action.md:74]:
```suggestion
          uses: project-copacetic/copa-action@v1
```

File: website/docs/github-action.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+          with:
+            image: ${{ matrix.images }}
+            image-report: 'report.json'
+            patched-tag: 'patched'
+            buildkit-version: 'v0.11.6'
+            # optional, default is latest
+            copa-version: '0.3.0'
```
Comment (sozercan) [inline website/docs/github-action.md:81]:
```suggestion
            copa-version: '0.4.1'
```

File: website/docs/github-action.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+        - name: Copa Action
+          if: steps.vuln_count.outputs.vuln_count != '0'
+          id: copa
+          uses: project-copacetic/copa-action@v1.0.0
+          with:
+            image: ${{ matrix.images }}
+            image-report: 'report.json'
+            patched-tag: 'patched'
+            buildkit-version: 'v0.11.6'
```
Comment (sozercan) [inline website/docs/github-action.md:79]: is this optional now?

File: website/docs/github-action.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
+# Copa Github Action
```
Comment (sozercan) [inline website/docs/github-action.md:1]: i think this needs metadata at the top, see existing docs

**Review by salaxander — APPROVED** [review]

**Pull request merged**